separate out existing tools (#168)
intel/he-toolkit: kit/tools/README.md

# HE Tools

> The tools commands for `hekit` contain useful commands to finds HE parameters

Reviewer: ```suggestion
The tools for `hekit` consist of useful commands for finding HE parameters
```

> based on user constraints. Next sections describe the details about them.

Reviewer: ```suggestion
based on user constraints. The following sections describe their details and usage.
```

## Dependencies
- python >= 3.8

## gen-primes

The command `hekit gen-primes` generates a list of sorted primes in range [n,

> m] where n and m are positive integers

Reviewer: ```suggestion
m] where n and m are positive integers.
```
